libs/wingc/src/type_check/jsii_importer.rs

```diff
@@ -354,6 +411,11 @@ impl<'a> JsiiImporter<'a> {
 				arg_types.push(wing_type);
 				// Define the rest of the arguments and create the method signature
 				if let Some(params) = &m.parameters {
+					if self.has_variadic_parameters(params) {
+						debug!("Skipping method {} with variadic parameters", m.name);
```

Comment:
Tracking issue https://github.com/winglang/wing/issues/397 or https://github.com/winglang/wing/issues/125.
Emit compiler wanrinig?

Reply:
Added links to issues